src/handlers.cpp

```diff
@@ -2548,6 +2550,28 @@ void PushProfileTask::update_reg_data()
     _impus = _irs_impus;
   }
 
+  if (_ims_sub_present && is_first_irs())
+  {
+    HTTPCode rc = HTTP_OK;
+    find_impus_to_delete();
+
+    rc = _cfg->sprout_conn->change_associated_identities(_default_public_id,
+                                                         _ims_subscription,
+                                                         this->trail());
+
+    if (rc == HTTP_OK)
+    {
+      TRC_DEBUG("Informed Sprout, return code %d", rc);
+    }
+    else
+    {
+      TRC_DEBUG("Sprout return Code %d, Send Push Profile answer indicating failure", rc);
```

Comment:
Weird capitalisation here - How about 'Failed to update Sprout (return code: %d), sending negative PPA'.